### ISSUE TITLE
Can't get the correct error rates

### ISSUE THREAD

**Issue by gagb** [musyoku/adversarial-autoencoder#7]
Hey

For the semi-supervised version (100 labels), the current version gets around 85% accuracy.
Which is different from the graph in the readme. 

Here is the resultant semi-supervised/dim_reduction/results.csv
[result.txt](https://github.com/musyoku/adversarial-autoencoder/files/859491/result.txt)

The performance of the unsupervised dimensionality reduction also seems to have decreased.
Here's the visualization using the current version. The separation is worse than the visualization shown in readme. The error rate on the training data for this visualization is ~40%. The error rate reported in the original paper is ~13.5%.
![labeled_z_test_default](https://cloud.githubusercontent.com/assets/13227607/24178016/9095dfb8-0e63-11e7-8d58-6cc9ad777e50.png)

Thanks!

**Comment by musyoku** [comment]
Hi.
I found that the default hyperparameter (`learning_rate`=0.001, `momentum`=0.5) is too high for semi-supervised / unsupervised learning.

### /semi-supervised/classification/

`learning_rate`=0.001, `momentum`=0.9

![image](https://cloud.githubusercontent.com/assets/15250418/24250450/4ecc4834-101a-11e7-82b1-a418fb5cb303.png)

`learning_rate`=0.001, `momentum`=0.1

![image](https://cloud.githubusercontent.com/assets/15250418/24250455/51faeba0-101a-11e7-9cba-777b59fd4d90.png)

`learning_rate`=0.0001, `momentum`=0.1

![image](https://cloud.githubusercontent.com/assets/15250418/24250460/534d847c-101a-11e7-906a-ff9d8180c6ea.png)

### /unsupervised/dim_reduction/

`learning_rate`=0.0001, `momentum`=0.1, `seed`=2

![2](https://cloud.githubusercontent.com/assets/15250418/24250521/8213ab9c-101a-11e7-9b32-3d9cc33f632f.png)

`learning_rate`=0.0001, `momentum`=0.1, `seed`=3

![3](https://cloud.githubusercontent.com/assets/15250418/24250534/8d805282-101a-11e7-8a71-fc0654d5c95d.png)

`learning_rate`=0.0001, `momentum`=0.1, `seed`=4

![4](https://cloud.githubusercontent.com/assets/15250418/24250549/9a5d89fc-101a-11e7-8207-4cead62d1640.png)

I fixed the code.

Thank you :smiley:

**Comment by gagb** [comment]
The unsupervised version now gets ~28.5% classification error. Thanks!
I am closing this issue for now because the lower performance seems to be because of choice of hyper-parameters and not because of an implementation error.
